gengo/languages.yaml

```diff
@@ -446,6 +446,14 @@ OCaml:
     extensions:
       - ml
       - mli
+Odin:
+  category: programming
+  color: "#3882D2"
+  matchers:
+    extensions:
+      - odin
+    interpreters:
+      - odin
```

Comment:
Can Odin be used as an interpreter? Some quick searching didn't turn anything substantial up. This entry is used for executable text files in Unix-like systems, where a file extension might not exist. For example:
```python
#!/usr/bin/env python
print('hello, world!')
```
```shell
#!/bin/sh
echo "hello, world!"
```

Reply:
oh my bad, from how the docs were worded I thought the interpreters field was for just associated programs, so i put the odin compiler. That can be taken out, then

Reply:
Thanks for the feedback! Perhaps I should make the docs more specific :thinking: